walkthroughs/week-3-server/examples/favorite-color/pom.xml

```diff
@@ -28,9 +28,13 @@
   <build>
     <plugins>
       <plugin>
-        <groupId>com.google.appengine</groupId>
+        <groupId>com.google.cloud.tools</groupId>
         <artifactId>appengine-maven-plugin</artifactId>
-        <version>1.9.71</version>
+        <version>2.2.0</version>
+        <configuration>
+          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
```

Comment:
Are these changes to the example projects intentional? They still don't have the project id specified so I can't imagine this actually works

Reply:
These changes are from the main STEP repository, so I think we as the interns add the project id to the file as we go through the trainings.

Reply:
Ah, gotcha. So you intentionally pulled these in from the STEP repo? That makes sense, in the future I'd prefer that be a separate PR though, one that doesn't also include your own changes.

Reply:
I will definitely make sure you pull in any future changes in a separate PR next time, sorry about that!